client/chat/channel.jsx

```diff
@@ -71,6 +71,18 @@ const UserSublist = styled.ul`
   list-style-type: none;
   margin: 0;
   padding: 0;
+
+  ${props => {
+    if (props.sectionTitle === 'Offline') {
```

Comment:
This *works* but it's pretty hacky imo. At the very least I'd move the string comparison to the JS function and pass in a bool for `faded` or something here